dpl-docs/views/layout.dt

```diff
@@ -6,6 +6,7 @@ html(lang='en-US')
     | http://digitalmars.com
   - import std.process : environment;
   - string version_id = environment["GIT_TARGET"];
+  - bool noExactSourceCodeLinks = environment["NO_EXACT_SOURCE_CODE_LINKS"] == "1";
```

Comment:
Apparently that's the "workaround" to inject parameters into the layout file - it's also the prevalent style here (see above).